### ISSUE TITLE
ConvertArray attribute bug

### ISSUE THREAD

**Issue by volodymyr-hordiienko** [yzen-dev/plain-to-class#11]
Hi! First of all just want to thank you for handful library! There is issue i found, documentaton says array of class items can be described using ConvertArray

```
class PurchaseDTO
{
    #[ConvertArray(ProductDTO::class)]
    public array $products;
}
```

But ConvertArray attribute is a #[Attribute(Attribute::TARGET_PARAMETER)] so it cant be assigned on property. Could TARGET_PROPERTY be added also?

**Comment by yzen-dev** [comment]
Hi! Thank you very much. Yes, it's true, at the start, I probably figured out the attributes badly

**Comment by volodymyr-hordiienko** [comment]
@yzen-dev could this change be released? I need it in my project :) I created fork and package for me currently but would be good to switch back on original library

**Comment by yzen-dev** [comment]
@volodymyr-hordiienko Hi! 
I didn't have time to make a release right away because of work. Now released 3.0.5 
https://github.com/yzen-dev/plain-to-class/releases/tag/v3.0.5

Can install:)
